src/components/Footer.js

```diff
@@ -1,6 +1,13 @@
+import Link from 'next/link';
+
 export default function Footer() {
 	return (
 		<footer>
+			<nav>
+				<Link href="/">Home</Link>
+				<Link href="/about">About me</Link>
+				<Link href="/bar">CLICK HERE !! my Bar !! CLICK HERE</Link>
+			</nav>
```

Comment:
This looks good! Just one remark for your copyright: You can use the html element `small` for copyrights. This would be more semantic than a `div`. [See MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/small).